### PR TITLE
PWA-18 add cf bind-service support

### PR DESCRIPTION
**Pull request by AustinMatherne**

### CODE REVIEW

File: rolling-deploy.sh [hunk 1]
```diff
@@ -44,14 +44,16 @@ declare json_file="${1}"
...
 
 if [[ $is_debug == true ]]; then
 	echo "${CF_API_ENDPOINT}"
-    echo "${CF_BUILDPACK}"    
+    echo "${CF_BUILDPACK}"
 	echo "${CF_USER}"
 	echo "${CF_PASSWORD}"
 	echo "${CF_ORG}"
 	echo "${CF_SPACE}"
+	echo "${CF_SERVICES}"
```
Comment (kaltepeter) [inline rolling-deploy.sh]: This won't print all values, only first one. Try this: `echo "${CF_SERVICES[@]}"`

File: rolling-deploy.sh [hunk 2]
```diff
@@ -79,6 +81,10 @@ cf push "${APP_NAME}-${BUILD_NUMBER}" -i 1 -m ${APP_MEMORY} \
   -b "${CF_BUILDPACK}" \
   -p "${ARTIFACT_PATH}" ${PUSH_OPTIONS}
 
+for CF_SERVICE in $CF_SERVICES; do
```
Comment (kaltepeter) [inline rolling-deploy.sh]: For bash array the syntax to get all values is:
```
for CF_SERVICE in "${CF_SERVICES[@]}"; do
...
done
```

**Comment by kaltepeter** [comment]
One more thing. I will also need you to bump the version in VERSION file.

**Comment by AustinMatherne** [comment]
Updated. Thanks, @kaltepeter!

File: cf-cli.sh [hunk 3]
```diff
@@ -39,6 +39,7 @@ declare json_file="${1}"
 # set cf vars
 read -r CF_API_ENDPOINT CF_USER CF_PASSWORD CF_ORG CF_SPACE CF_INTERNAL_APPS_DOMAIN CF_EXTERNAL_APPS_DOMAIN <<<$(jq -r '.cf | "\(.api_endpoint) \(.user) \(.password) \(.org) \(.space) \(.apps_domain.internal) \(.apps_domain.external)"' "${json_file}")
 read -r APP_NAME APP_MEMORY ARTIFACT_PATH BUILD_NUMBER EXTERNAL_APP_HOSTNAME PUSH_OPTIONS <<<$(jq -r '. | "\(.app_name) \(.app_memory) \(.artifact_path) \(.build_number) \(.external_app_hostname) \(.push_options)"' "${json_file}")
+read -r CF_SERVICES <<<$(jq -r '.cf.services[]' "${json_file}")
```
Comment (kaltepeter) [inline cf-cli.sh]: this would be the same feedback as below on readarray, sorry i missed this the first time.

**Pull request merged**